counterpartylib/test/complex_unit_test.py

```diff
@@ -0,0 +1,141 @@
+import pprint
```

Comment:
this is the template 'complex unittest' which demostrates the use of the mocked UTXO set and is a good base for any tests that aren't just 1 function call (which almost all our current vectors are).